tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,105 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    Click Link    Resources
+    Sleep    5s
+    Click Element    //div[@class="pf-c-toolbar__content-section"]/div[2]/div/button
+    Click Button    //button[@data-key="name"]
+    Sleep    1s
+    ${list_of_tile} =    Get List Of Ids Of Tiles
```

Comment:
I'm having problems to understand what this test is doing without checking  Polarion. Could you refactor the main test to use simple keywords (but meaningfully named) wrapping the actual code?

Reply:
Done